spark_connect_labextension/handlers/base.py

```diff
@@ -0,0 +1,94 @@
+"""
+Base API Route Handler
+
+This file contains the base class inherited by all API route handlers, 
+which provides common property for accessing the extension configuration.
+"""
+
+import json
+from jupyter_server.base.handlers import APIHandler
+from jupyter_server.extension.handler import ExtensionHandlerMixin
+from jsonpath_ng import jsonpath, parse
+
+
+class SparkConnectAPIHandler(ExtensionHandlerMixin, APIHandler):
+    @property
+    def ext_config(self):
```

Comment:
What does this configuration contain that is not related to clusters, bundles, spark options and error messages?

Reply:
Nothing, this method is to be invoked by the other properties (e.g. `spark_clusters`)

Reply:
Yes, will do